### PR TITLE
Unify containers and labels

### PR DESCRIPTION
**Pull request by attitude**
## Unifies naming of the containers to their purpose

* `FormGroup` to `FieldContainer` because it contains one input with label
* `BoxSection` to `RepeaterContainer` because it own only to the `Repeater`
* `BoxContent` merged with `Box` (reverts splitting & moves heading and actions "inside the box")
* `FieldLabel` merged with `Label`

**Important:** _Includes hotfix to resolve horizontal scrolling causing sticky position of hovering toolbar to break._

#### Added to Storybook

* `FieldContainer`
* `Divider`
* `Spacer`

### Unifies spacing of

  * `Stack`
  * `Divider`
  * `Spacer`
  * `Box`

### Fixes `Checkbox` and `Radio`

- Both now reuse `FieldContainer`
- responsibility to show errors was moved back to `FieldContainer` as with other inputs, e.g. `TextField`

Closes https://github.com/contember/private-issues/issues/8

Also introduces mock objects to use in Stories. Stories now use `<Select />` to change themes to save space instead of `<Radio />`.

---

### Checks
- [x] The changes and implementation strategy have been consulted with the maintainers.
- [x] The code passes all the checks (Run `pnpm run check`).

### CODE REVIEW

File: packages/ui/stories/src/RepeaterContainer.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	},
+} as ComponentMeta<typeof RepeaterContainer>
+
+const Template: ComponentStory<typeof RepeaterContainer> = args => <RepeaterContainer {...args} />
+
+export const Defaut = Template.bind({})
+
+Defaut.args = {
+	children: <>
+		<div>{'[input component]'}</div>
```
Comment (VojtaStanek) [inline packages/ui/stories/src/RepeaterContainer.stories.tsx:24]: Why not?

```suggestion
		<div>[input component]</div>
```

File: packages/ui/src/styles/components/editorCanvas.sass [hunk 2]
```diff
@@ -106,6 +105,7 @@
 	&.view-seamless
 		box-shadow: none
 
-	& &
-		&:not(:first-child)
-			margin-top: var(--cui-formGroup-separation)
+	// TODO: Maybe remove completely because --var is missing and has no effect
+	// & &
+	// 	&:not(:first-child)
+	// 		margin-top: var(--cui-fieldContainer-separation)
```
Comment (VojtaStanek) [inline packages/ui/src/styles/components/editorCanvas.sass:111]:
```suggestion
```

**Comment by VojtaStanek** [comment]
> `BoxSection` to `RepeaterContainer` because it own only to the Repeater

This component is a container of a single item, isn't it? For me the new name suggests it is a container of the whole repeater, not only of a single item. 

I would propose name `RepeaterSection`.

**Comment by attitude** [comment]
`RepeaterItemContainer` would make more sense if we can agree. Sections might be confused with section as in HTML specs.

**Pull request merged**